content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods.rst

```diff
@@ -0,0 +1,103 @@
+===================
+
+Companies that sell and deliver goods to customers need to make sure they always have stock on-hand,
+so when new sales orders are confirmed, they can deliver products on time.
+
+In Odoo, this can be handled using *reservation methods*. Reservation methods control how products
+included in a delivery order (DO) should be reserved for delivery, ensuring they are reserved at the
+correct times, for the correct orders.
+
+There are three different reservation methods in Odoo: *At Confirmation*, *Manually*, and *Before
+scheduled date*.
+
+.. tabs::
+
+   .. tab:: At Confirmation
+
+      Reserves products **only** when a sales order is confirmed, **and** if stock is already
+      available.
+
+   .. tab:: Manually
+
+      Once a quote is confirmed, product availability **must** be checked manually, and the required
+      quantity **must** be reserved manually.
+
+   .. tab:: Before scheduled date
+
+      A specific number of days can be selected; this is the maximum number of days **before** a
+      scheduled delivery date that products should be reserved.
+
+Configuration
+=============
+
+Reservation methods are set on individual operations types. To configure reservation methods, go to
+:menuselection:`Inventory app --> Configuration --> Operations Types`. Then, select the desired
+operation type. Or, create a new one by clicking :guilabel:`New`.
+
+In the :guilabel:`General` tab of the operation type form, locate the :guilabel:`Reservation Method`
+option, and choose which method should be used for this type of operation.
+
+.. image:: reservation_methods/reservation-methods-operations-type-field.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+.. tip::
+   If the :guilabel:`Before scheduled date` reservation method is selected, a new
+   :guilabel:`Reserve before scheduled date` field appears below. From this field, the number of
+   :guilabel:`days before` and :guilabel:`days before when starred` can be changed from the default
+   `0`.
+
+   Changing the :guilabel:`days before` value changes the maximum number of days before a scheduled
+   date that products should be reserved.
+
+   Changing the :guilabel:`days before when starred` value changes the maximum number of days before
+   a scheduled date that starred (favorited) transfers for products should be reserved.
+
+   .. image:: reservation_methods/reservation-methods-before-scheduled-date.png
+      :align: center
+      :alt: Reserve before scheduled date fields with before scheduled date method chosen.
+
+Required applications
+=====================
+
+The two required applications that **must** be installed to use reservation methods are the *Sales*
+and *Inventory* apps.
+
+To install these apps, from the main Odoo dashboard, navigate to the :menuselection:`Apps`
+application, and locate the *Sales* and *Inventory* apps. Then, click :guilabel:`Install` on each
+app, and refresh the database.
```

Comment:
Suggesting to use the **Apps and modules Install** documentation instead of re-explaining how to install the modules :)
```suggestion
The two required applications that **must** be :ref:`installed <general/install>` to use reservation
methods are the *Sales* and *Inventory* apps.
```

Reply:
That's a nice way to save some time! Suggestion implemented ✅